contracts/FastBreakV1.cdc

```diff
@@ -122,13 +122,136 @@ access(all) contract FastBreakV1: NonFungibleToken {
     }
 
     /// Metadata Dictionaries
+    /// Player mappings remain in contract storage (bounded by number of users)
+    /// Old dictionaries kept for existing data (pre-upgrade)
     ///
-    access(self) let fastBreakRunByID:      {String: FastBreakRun}
-    access(self) let fastBreakGameByID:     {String: FastBreakGame}
+    access(self) let fastBreakRunByID:      {String: FastBreakRun}  // Legacy: existing data
+    access(self) let fastBreakGameByID:     {String: FastBreakGame}  // Legacy: existing data
     access(self) let fastBreakPlayerByID:   {UInt64: PlayerData}
     access(self) let playerAccountMapping:  {UInt64: Address}
     access(self) let accountPlayerMapping:  {Address: UInt64}
 
+    /// Storage resources for year-based organization
+    /// Each year's games/runs stored in separate resources in account storage
+    ///
+    access(all) resource YearGameStorage {
+        access(self) var games: {String: FastBreakGame}
```

Comment:
Could be helpful to add  a `access(self) var extension: {String: {String: AnyStruct}}` map just in case anything else needs to be stored in that resource in the future. Same for `YearRunStorage`